test/tsc-installer.js

```diff
@@ -0,0 +1,22 @@
+const { execSync } = require('child_process');
+const { join } = require('path');
+const { readdirSync, lstatSync } = require('fs');
+
+const tscListDir = join(__dirname, 'tsc-list/');
+
+class TscInstaller {
```

Comment:
There's no internal state to this class, so you might want to change it to functions only?

Reply:
Good idea